app/src/main/java/org/kiwix/kiwixmobile/zim_manager/fileselect_view/ZimFileSelectFragment.java

```diff
@@ -194,18 +195,14 @@ public boolean onActionItemClicked(ActionMode mode, MenuItem item) {
                       selectedViewPosition.get(i));
               String shareFilePath = data.file.getPath(); //Returns path to file in device storage
 
-              /**
```

Comment:
Do not delete comments there must be an option to change the docs comment to block comment you must have used that.